tests/flow/test_map.py

```diff
@@ -54,6 +54,35 @@ def test02_map_projections(self):
                            [{'val': 3, 'literal': 'lit'}]]
         self.env.assertEquals(query_result.result_set, expected_result)
 
+        # map projection unexisting property
```

Comment:
Can we add a test for nested projection?
e.g. `MATCH (a), (b) RETURN a{.*, b {.*} }`, `MATCH (a) RETURN a{.*, c {.*} }`

Reply:
Ok. Tests added. [69c4553](https://github.com/RedisGraph/RedisGraph/pull/2698/commits/69c4553285e66fb2123936b1b7be73aa5b4dcf88)
Nested projections returns invalid input error, the same result is obtained using neo4j